lib/streamlit/commands/query_params.py

```diff
@@ -58,7 +58,7 @@ def get_query_params() -> Dict[str, List[str]]:
         return {}
     # Return new query params dict, but without embed, embed_options query params
     return util.exclude_key_query_params(
-        parse.parse_qs(ctx.query_string), keys_to_exclude=EMBED_QUERY_PARAMS_KEYS
+        parse.parse_qs(ctx.query_string, True), keys_to_exclude=EMBED_QUERY_PARAMS_KEYS
```

Comment:
Can the new argument be passed as a `kwarg` so that the code is more readable / self-documenting?

Reply:
Totally makes sense. thanks for that.